crates/re_ui/Cargo.toml

```diff
@@ -26,6 +26,9 @@ all-features = true
 default = []
 
 [dependencies]
+re_entity_db.workspace = true # syntax-highlighting for InstancePath. TODO(emilk): move InstancePath
```

Comment:
more intance path to where?

aren't these making re_ui a lot less lightweight?

Reply:
> move instance path to where?
It feels like it belongs in a more low-level crate, like `re_types`

> aren't these making re_ui a lot less lightweight?

Yes. The alternative is to create yet another crate, or add more feature flags, and both kind of sucks.